.pre-commit-test.yaml

```
@@ -6,6 +6,6 @@ repos:
       entry: mdformat
       files: "tests/pre-commit-test.md"
       types: [markdown]
-      language: system
+      language: python
       additional_dependencies:
         - mdformat-plugin
```

I think removing `additional_dependencies` and keeping `language` as is is the correct fix here
```suggestion
      language: system
```

---

I'm against changing to `system`, this means it would only work from a pre-configured python environment and not in an automatically generated environment. It stops `pre-commit install` and `pre-commit/action@v2.0.0` from working (and also things like https://pre-commit.ci/)

---

Note that that would not be a change. It was `system` and was updated to `python` in this PR. All I'm suggesting is to keep that.

Also note that this is `.pre-commit-test.yaml ` not `.pre-commit-config.yaml`. My understanding is that this file exists so that we can test pre-commit hook with `mdformat` and the plugin in its current state in this repo (unreleased version). That is something we can't install via `additional_dependencies` (due to not being released yet).

I might be wrong though, not 100% what the intent of this file was originally.

---

ah yeh thats alright, ignore me then